src/components/movie-view/movie-view.jsx

```diff
@@ -32,7 +32,13 @@ export const MovieView = ({ movie, onBackClick }) => {
         <span>Release Year: </span>
         <span>{movie.ReleaseYear} </span>
       </div>
+      <div>
+        <span>Featured: </span>
+        <span>{movie.Featured}</span>
```

Comment:
Because `movie.Featured` is a boolean, it's not disabled in html. To display it, we need to explicitly decide what string to display for each case ie when its value is `false`, and when its `true`

```suggestion
        <span>{movie.Featured ? "Yes" : "No"}</span>
```

Reply:
Oh that's handy, the template literal is a good solution.  Thanks for the tip.